src/registry.ts

```diff
@@ -164,6 +165,9 @@ export function getEntryUrl(
     ...customOptions,
   };
 
+  // Trim the prefix if it was passed in.
+  publicKey = trimPrefix(publicKey, "ed25519:");
```

Comment:
If worried about URL encoded public key passed in, using `decodeURI(publicKey)`.

Reply:
I would only accept un-encoded input, trying to decode as a URI here doesn't feel right. But I can add a check for valid hex-encoded input and throw if not -- this would catch any unexpected prefixes or encodings (since `:` and `%` are not hex characters) and the user would be notified of the problem.